CDTDatastoreTests/CDTSessionCookieInterceptorTests.m

```diff
@@ -39,6 +47,11 @@ @implementation CDTSessionCookieInterceptorTests
 
 - (void)setUp
 {
+    setenv("CDT_TEST_ENABLE_OHHTTPSTUBS", "1", true);
```

Comment:
Should unset this in the teardown, otherwise the background session won't be used for all network requests after these tests. It's not a big deal but our tests should be as close to production usage as possible.